activiti-services/activiti-services-registry/pom.xml

```diff
@@ -33,6 +33,16 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
+      <exclusions>
```

Comment:
@ryandawsonuk why do we need jetty here? I mean, if we don't have the keycloack adapter